requirements.txt

```diff
@@ -1 +1,3 @@
-omego
\ No newline at end of file
+omego
+reportlab
```

Comment:
we may have some surprise as we had with pillow if we install from pip
That's why we removed them from the installation doc